web/src/buddy_data.ts

```diff
@@ -465,7 +465,7 @@ function get_filtered_user_id_list(
     const user_ids_set = new Set([...base_user_id_list, ...conversation_participants]);
     return filter_user_ids(user_filter_text, [...user_ids_set]);
 }
-
+// get participants of the current viewed conversation.
```

Comment:
You use the phrase "No.participants" in the commit message, presumably an abbreviation of "number of participants". We have a policy against using abbreviations like that in Zulip commit messages; it makes things harder to read and typing is fast, so it doesn't have much benefit.

Reply:
done